### ISSUE TITLE
Unnecessary dialog shown by Google SmartLock

### ISSUE THREAD

**Issue by Nilzor** [schibsted/account-sdk-android#393]
Disclaimer: I don't know the code well so I don't know if this is fixable

It looks like Google SmartLock shows an unnecessary and annoying dialog at the start of the singup flow in the following case:

- The user have configured the phone with multiple GMail accounts
- The user hasn't stored the password for the Schibsted Account in any of those accounts

The dialog looks like this

![image](https://user-images.githubusercontent.com/990654/60501421-51b34a80-9cbc-11e9-99d5-2ba8bcc76d6a.png)

It basically asks us to choose which account "to continue with". I don't understand what selecting any of those actually _does_ though. 

When done entering a username and password, we get aske once again which account to store it in. _This dialog_ is necessary and fine:

![image](https://user-images.githubusercontent.com/990654/60501729-c5555780-9cbc-11e9-944d-91a9af60aef5.png)

Finally note that when you _do_ have a password stored in any of the configured accounts, the initial dialog int he flow looks like this (and is also fine)

![image](https://user-images.githubusercontent.com/990654/60501679-aeaf0080-9cbc-11e9-8616-33b1d1830e00.png)

In summary: I want someone to find out if we can drop the dialog illustrated at the top here, and still retain all the functionality of Google SmartLock. It would improve the user experience

**Comment by Nilzor** [comment]
Have you had a chance to look at this?

**Comment by zamzterz** [comment]
Yes, I finally got around to it now after vacation:
The initial prompt is expected, see step 4 [here](https://developers.google.com/identity/smartlock-passwords/android/overview), but it's supposed to result in the selected identifier being used/pre-filled in the login/signup flow. That was not working correctly, but has been fixed in #404.
